debug_toolbar/toolbar.py

```diff
@@ -72,6 +73,17 @@ def csp_nonce(self):
         """
         return getattr(self.request, "csp_nonce", None)
 
+    @property
+    def health_level(self):
+        """
+        Return the maximum health level across all panels.
+        This is used to color the toolbar hidden button.
+        """
+        if not self.panels:
+            return HealthLevel.NONE
+
+        return max(panel.health_level for panel in self.panels)
```

Comment:
```suggestion
        return max(panel.health_level for panel in self.enabled_panels)
```
We should only try to fetch the value for the panels that are turned on.